app/src/main/res/drawable/ic_baseline_chat.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:autoMirrored="true" android:height="50dp"
+    android:tint="#A786E3" android:viewportHeight="24"
+    android:viewportWidth="24" android:width="50dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="@android:color/white" android:pathData="M20,2L4,2c-1.1,0 -1.99,0.9 -1.99,2L2,22l4,-4h14c1.1,0 2,-0.9 2,-2L22,4c0,-1.1 -0.9,-2 -2,-2zM6,9h12v2L6,11L6,9zM14,14L6,14v-2h8v2zM18,8L6,8L6,6h12v2z"/>
```

Comment:
Is it possible to extract the pathData to a file? I suppose it's an icon, it could be good to separate it from the layout

Reply:
I actually tried this, to first convert xml file to svg and then from svg to png but I had errors. I don't think it's a big deal so I'll leave this as it is.